### ISSUE TITLE
we should be able to do dotted gets on a new invocation

### ISSUE THREAD

**Issue by georgejecook** [rokucommunity/brighterscript#595]
e..g       
`item = new mc.collections.Util(m).filter(items, function(i): return i = m.localData: end function)`
or
`appId = new Appsettings().appid`

or 
`? new Appsettings().appid`

or 
`uCase(new Appsettings().appid)`

etc

**Comment by TwitchBronBron** [comment]
Yeah, I think javascript supports this pattern as well. For now, you can work around it by using parens, but long-term we should definitely support this syntax.

```
appId = (new AppSettings()).appId
```
